Dockerfile

```diff
@@ -74,6 +74,6 @@ RUN curl -s https://letsencrypt.org/certs/lets-encrypt-x1-cross-signed.pem > /et
 ADD target/bin/generate-ssl-certificate target/bin/generate-dkim-config target/start-mailserver.sh /usr/local/bin/
 RUN chmod +x /usr/local/bin/*
 
-EXPOSE 25 587 143 993 110 995
+EXPOSE 25 587 143 993 110 995 4190
```

Comment:
Yes it needs to be exposed because the client wants to connect to the manage sieve daemon and edit the sieve files online.